node/controllers/emailVerification/index.js

```diff
@@ -11,7 +11,7 @@ export const handleEmailVerificationRequest = async (req, res) => {
     const user = await User.findOne({ where: { email } })
     if (user.emailVerified) {
       return res.status(201).send({
```

Comment:
Hmm, this is kind of misleading, since status `201` is "created" and implies that request was infact successful. I think either we should change the status code or success, depending on how we will handle this request in the frontend.